Manufacturers.md

```diff
@@ -23,6 +23,7 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |DIAT|Diatone Innovations|https://www.diatoneusa.com/|
 |DRCL|dronercland|https://www.instagram.com/dronercland/|
 |DYST|DongYang Smart Technology Co.,Ltd (dys)|http://www.dys.hk/|
+|EACH|eachine|https://www.eachine.com/|
```

Comment:
Eachine must not be uppercase?

Reply:
Of course.
 "|EACH|EACHINE|https://www.eachine.com/|"

Reply:
I got this from the copyright notice on the official website - which is where I normally go for the official company name:

![image](https://user-images.githubusercontent.com/4742747/87237296-b0231500-c448-11ea-8d94-349eae6929a9.png)

Looking at https://www.eachine.com/About-Us-a-1.html, there seems to be a mix of casings used across the page, so not sure what to use:

![image](https://user-images.githubusercontent.com/4742747/87237306-d2b52e00-c448-11ea-9c76-2c287a78b474.png)

Will go with what we have for now, can always update when this has been cleaned up.